verification/verify.go

```diff
@@ -103,7 +112,29 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 		return nil, nil
 	}
 
-	// TODO sig checking; extract from broadcast.go
+	if params.Results != nil {
+		segHashes := make([][]byte, len(params.Results.Segments))
+		for i, segment := range params.Results.Segments {
+
+			data, err := drivers.GetSegmentData(segment.Url)
+			if err != nil {
+				glog.Errorf("%v error with segment: %v (URL: %v)", monitor.SegmentTranscodeErrorDownload, err, segment.Url)
```

Comment:
Could we just return early with the error here? This would mean that if we can't download the data in order to verify the signature then we return an error.

Reply:
I agree.  However, we were able to remove this block of code in 4def93a69fdbc03ac0623ab0ece10451070649a9 , so this particular comment no longer applies.